### PR TITLE
Reboot comment

### PR DESCRIPTION
**Pull request by TinaBond**
Reboot comment (from first closed PR)

### CODE REVIEW

**Comment by TinaBond** [comment]
Maybe... 
In the future, describing rules for size and format of images, gif`s etc. will be better. 😊

**Comment by TinaBond** [comment]
fixed conflict

**Comment by suchov** [comment]
@TinaBond I'm a bit confused. 68 files changed in your pr... Is it done on purpose?

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-minimal
```
Comment (suchov) [inline _config.yml:1]: Is this done on purpose?

File: README.md [hunk 2]
```diff
@@ -3795,13 +3808,15 @@ My name is [Zarema](https://github.com/ZaremaK/kottans-frontend). I'm glad to jo
...
 
+
 ## Hello world! :D
 
 ## Hi, Kottans!
 ### I'm Valentyn. 
 My  [Repository for Kottans Front-End](https://github.com/Chotogde/kottans-frontend).
 :round_pushpin: don't forget to `cd mock-repo` in concole after cloning rerpository kottans/mock-repo :sweat_smile:
 
+
```
Comment (suchov) [inline README.md:3819]: do we need this extra lines?

**Review by suchov — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (TinaBond) [reply _config.yml:1]: I don't know, unfortunately, how it was added. 😕

File: README.md (hunk 2 above)
Comment (TinaBond) [reply README.md:3819]: No, there don't need.

**Comment by suchov** [comment]
So you changed 68 files and how many lines of code?

**Comment by TinaBond** [comment]
@suchov In first PR I did resize images. Then I changed (reboot) my commit. (I think so).
Did something else come out as a result?

**Comment by TinaBond** [comment]
@suchov  what should I'm doing now to fix it?

**Comment by suchov** [comment]
I actually like that you made all the images smaller.(all of them)
Just check the link https://github.com/kottans/mock-repo/pull/977/files. These are all the changes that you introduced. 
In general, I don't see any problems with merging it.

**Comment by TinaBond** [comment]
> I actually like that you made all the images smaller.(all of them) Just check the link https://github.com/kottans/mock-repo/pull/977/files. These are all the changes that you introduced. In general, I don't see any problems with merging it.

:slightly_frowning_face: 
I'm sorry, but I understand, what I need to check that file  :slightly_frowning_face:
And what ["Changes requested" is yellow?](https://nimb.ws/1nKB0f)

**Review by TinaBond — COMMENTED** [review]
I chek link

**Review by TinaBond — COMMENTED** [review]
I chek

**Review by TinaBond — COMMENTED** [review]
Changes have been made

**Review by suchov — APPROVED** [review]

**Comment by suchov** [comment]
Good work.

**Comment by TinaBond** [comment]
Thanks, @suchov 😸

**Pull request merged**